samples/convert-upload-images-powervs/convert-upload-images-powervs

```diff
@@ -0,0 +1,742 @@
+#!/usr/bin/env bash
+: '
+Copyright (C) 2021 IBM Corporation
+Licensed under the Apache License, Version 2.0 (the “License”);
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an “AS IS” BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+'
+#-------------------------------------------------------------------------
+set -e
+#set -x
+
+source <(curl -L https://raw.githubusercontent.com/ocp-power-automation/openshift-install-power/master/openshift-install-powervs 2> /dev/null | sed  's/main "$@"//g')
```

Comment:
Please source a specific tag or commit version to reduce the dependency on the called script.